File: CH1-input-process-output/EX8-tip-tax-total/tipTaxTotal.cpp
#include <iostream>
using namespace std;

int main()
{
    int total;
    int charge;
    float tip = 0.15;
    float totalTip;
    float salesTax = 0.07;
    float totalSalesTax;

    // Input
    cout << "Charge Food : $";
    cin >> charge;

    // Process
    totalTip = charge * tip;
    totalSalesTax = charge * salesTax;
    total = charge + totalTip + totalSalesTax;

    // Output
    cout << "Charge    : $" << charge << endl;
    cout << "Tip       : $" << totalTip << endl;
    cout << "Sales Tax : $" << totalSalesTax << endl;
    cout << "Total     : $" << total << endl;

    return 0;
}

File: CH1-input-process-output/EX8-tip-tax-total/tipTaxTotal.py
# Input
tip = 0.15
salesTax = 0.07
charge = int(input("Charge Food : $"))

# Process
totalTip = charge * tip
totalTip = int(totalTip)
totalSalesTax = charge * salesTax
totalSalesTax = int(totalSalesTax)
total = charge + totalTip + totalSalesTax

# Output
print("Charge    : $" + str(charge))
print("Tip       : $" + str(totalTip))
print("Sales Tax : $" + str(totalSalesTax))
print("Total     : $" + str(total))
